lib/server/utils.js

```diff
@@ -128,3 +130,23 @@ function parseForm(request) {
     });
   });
 }
+
+const LEGACY_FORMAT_REGEX = /(.*)-(\d+)\.json$/;
```

Comment:
The regex will pass the new format as well:
![Screenshot 2022-01-03 at 10 49 52](https://user-images.githubusercontent.com/650918/147896678-2567b175-0299-4f33-86ec-43882b21a799.png)

Maybe something like this?

```suggestion
const LEGACY_FORMAT_REGEX = /((?:\w*-){1,}\w*)-(\d+)\.json$/;
```

Reply:
![Screenshot 2022-01-03 at 11 19 22](https://user-images.githubusercontent.com/650918/147897580-a457a429-6d4c-466a-b37f-9b8c37c7225f.png)